lib/gateway/Shard.js

```diff
@@ -445,7 +445,7 @@ class Shard extends EventEmitter {
                     * @prop {PrivateChannel | TextChannel} channel The text channel the user is typing in
                     * @prop {User} user The user
```

Comment:
```suggestion
                    * @prop {PrivateChannel | TextChannel} channel The text channel the user is typing in. If the channel is not cached, this will be an object with an `id` key. No other property is guaranteed.
                    * @prop {User} user The user. If the user is not cached, this will be an object with an `id` key. No other property is guaranteed.
```